packages/core/addon/components/navi-visualizations/line-chart.js

```diff
@@ -199,19 +200,40 @@ export default Component.extend({
   }),
 
   /**
-   * @property {Object} data - configuration for chart x and y values
+   * @property {Array} seriesData - chart series data
    */
-  dataConfig: computed('firstModel', 'seriesConfig', 'c3ChartType', function() {
+  seriesData: computed('firstModel', 'seriesConfig', function() {
     const request = get(this, 'firstModel.request');
     const rows = get(this, 'firstModel.response.rows');
     const builder = get(this, 'builder');
     const seriesConfig = get(this, 'seriesConfig.config');
-    const seriesData = builder.buildData(rows, seriesConfig, request);
+    return builder.buildData(rows, seriesConfig, request);
+  }),
+
+  /**
+   * @property {Array} seriesDataGroups - chart series groups for stacking
+   */
+  seriesDataGroups: computed('options', 'seriesData', function() {
+    const seriesData = get(this, 'seriesData'),
+      options = merge({}, DEFAULT_OPTIONS, this.options),
+      { stacked } = options.style;
+
+    return stacked && seriesData.length ? [Object.keys(seriesData[0]).filter(key => key !== 'x')] : [];
+  }),
+
+  /**
+   * @property {Object} dataConfig - configuration for chart x and y values
+   */
+  dataConfig: computed('c3ChartType', 'seriesData', 'seriesDataGroups', function() {
+    const c3ChartType = get(this, 'c3ChartType'),
```

Comment:
Can we do `this.{prop}`?

Reply:
`this.seriesData` breaks in Ember 2.x

Reply:
Hmm, did you test this?

Reply:
yes, but I'll take another look

Reply:
@jkusa you mean in #421?
Now that I merged it in I see that this, too, breaks in 2.x.
Looks like `this.{prop}` when using with cp's returns a `ComputedProperty` and not the value.
@kevinhinterlong have you tested it with 2.x? I might be missing something here.